tests/unit/scripts/test_importer.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
```

Comment:
I'm not as stringent about tests for scripts than I am for core qutebrowser code, but indeed it really makes sense to have some tests for this now. If you want to add some more (maybe just something with example sqlite files from Firefox/Chromium checking that the right thing comes out), that'd be much appreciated!